### PR TITLE
Fixing and cleaning indexing

### PR DESCRIPTION
**Pull request by D-Kuma-C**
Reworked indexing and search, so it works with the database. It possible to search for word in files.

### CODE REVIEW

File: src/Endpoint-Tester/package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "license": "GPL-2.0-only",
   "description": "",
   "dependencies": {
+    "endpoint-tester": "file:",
```
Comment (Klint02) [inline src/Endpoint-Tester/package.json]: Remove this

File: src/P7-PSEngine/API/IndexService.cs
Comment (Klint02) [inline src/P7-PSEngine/API/IndexService.cs:1]: Maybe place into a service folder

File: src/P7-PSEngine/Files/common_english_words.txt
Comment (Klint02) [inline src/P7-PSEngine/Files/common_english_words.txt:1]: gitignore

**Review by Klint02 — APPROVED** [review]
Look good

**Review by clahanse — APPROVED** [review]
Looks fine

**Pull request merged**